serde_derive/src/de.rs

```diff
@@ -1336,9 +1336,7 @@ fn deserialize_internally_tagged_enum(
                 params,
                 variant,
                 cattrs,
-                quote! {
-                    _serde::__private::de::ContentDeserializer::<__D::Error>::new(__tagged.content)
-                },
+                quote!(__deserializer),
```

Comment:
took me a moment, but this is only ever moved into one arm, but all arms move out of it. So this makes sense